### PR TITLE
Added Tile class with Value and BackgroundColor properties and test cases

### PR DESCRIPTION
**Pull request by adrielgamorim**

### CODE REVIEW

**Comment by gioacchinogioffre** [comment]
Good job! I'm thinking that we are gonna have more colors so we should think how we're gonna define them.

**Review by gioacchinogioffre — DISMISSED** [review]

File: Game2048/Game/Models/TileComponent.cs [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+                return "#fff";
+            }
+            else if (Value <= 3)
+            {
+                return "#aaa";
+            }
+            else if (Value <= 5)
+            {
+                return "#aa0";
+            }
```
Comment (Luisjor) [inline Game2048/Game/Models/TileComponent.cs:22]: Question, what happens with higher value tiles like 16, 32, 64, 128, 256...?

**Review by gioacchinogioffre — APPROVED** [review]
Good improvement! Maybe later we could even separate the color's dictionary in a Color single class to make it extendable and re usable.

File: Game2048/Game/Models/TileComponent.cs (hunk 1 above)
Comment (adrielgamorim) [reply Game2048/Game/Models/TileComponent.cs:34]: Question answered by voice chat

File: Game2048/Game/Models/TileComponent.cs [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+            {
+                colors.Add(Value.Value, System.Drawing.Color.FromArgb(random.Next(0, 256), random.Next(0, 256), random.Next(0, 256)));
+            }
+
+            System.Drawing.Color color = colors[Value.Value];
+            return "#" + color.R.ToString("X2") + color.G.ToString("X2") + color.B.ToString("X2");
+        }
+    }
+
+    public string Color
```
Comment (J0c4) [inline Game2048/Game/Models/TileComponent.cs:27]: suggestion: `NumberColor`

**Pull request merged**